src/float-counts-prune.cc

```diff
@@ -483,7 +483,7 @@ class FloatCountsPruner {
   // the count to the backoff state.  Later on we'll structurally
   // remove the pruned counts.
   void DoPruningForLmState(int32 history_length) {
-    if (history_length + 1 > num_ngrams_per_order_.size()) {
+    if (history_length + 1 > (int32)num_ngrams_per_order_.size()) {
```

Comment:
due to google style guide, we avoid C-style casts.  Please write as e.g.
  if (static_cast<size_t>(history_length + 1) > num_mgrams_per_order.size()) {